src/components/TaskEntry.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+TaskEntry.propTypes = {
+  nameOfCrop: PropTypes.string,
+  date: PropTypes.string,
+  tasks: PropTypes.string,
+  onClick: PropTypes.func.isRequired,
+}
+
+export default function TaskEntry({ onClick, date, nameOfCrop, tasks }) {
+  return (
+    <Wrapper>
+      <input type="checkbox" onClick={onClick} />
+      <Grid>
+        <h2>{formatDate(date)}</h2>
+        <Flex>
+          <span>{nameOfCrop}:</span>
+          <span>{tasks}</span>
```

Comment:
is this a list of tasks or just a single task? i would expect a list in an array and not in a string